transport/http/constants.go

```diff
@@ -22,20 +22,24 @@ package http
 
 const (
 	// CallerHeader is the HTTP header used to indiate the service doing the calling
-	CallerHeader = "RPC-Caller"
+	CallerHeader = "Rpc-Caller"
```

Comment:
We should be, although I'll bet that most of the http clients are not. Let's just be sure to use the same casing for our own sanity.